src/data/teams.ts

```diff
@@ -0,0 +1,65 @@
+import { Team } from "../types";
+
+export const Teams: Team[] = [
+  {
+    description:
+      "This team supports and enables everyone else to be able to do what needs to be done. They are also responsible for the high level decisions that chart the overall course of the project.",
+    members: [
+      ["ReillyBrogan", null],
+      ["JoeyRiches", null],
+      ["TraceyClark", null],
+      ["TroyHarvey", null],
+      ["DavidHarder", null],
+    ],
+    name: "Admin Team",
+  },
+  {
+    description:
+      "This team handles the building of packages in the Solus repository. Some team members help maintain one or more of our included desktop environments.",
+    members: [
+      ["JustinZobel", "Plasma"],
```

Comment:
Given that Justin has stepped back from Solus, and hadn't really done much with Plasma updates, we should probably remove the Plasma tag